fastai/dataset.py

```diff
@@ -47,6 +47,43 @@ def read_dirs(path, folder):
             labels.append(label)
     return filenames, labels, all_labels
 
+def create_sample(path, r):
+    """ Takes a path to a dataset and creates a sample of specified size at <path>_sample
+
+    Parameters:
+    -----------
+    path: dataset path
+    r (float): how many examples to use as a sample, in the range from 0 to 1
```

Comment:
Minor wording suggestion: 'proportion of examples to use as a sample, in the range from 0 to 1'?

Reply:
changed